src/client/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -43,9 +43,9 @@ class ErrorableTextInput extends React.Component {
     let errorSpanId = undefined;
     let inputErrorClass = undefined;
     let labelErrorClass = undefined;
-    if (this.props.errorMessage) {
+    if (this.props.errorMessage || this.props.field.value.length > this.props.charMax) {
```

Comment:
@crwallace I totally agree. Let's use `maxLength`. Can we still show that message below the input (what you originally had) conditionally, though, on `this.props.field.value.length >= this.props.charMax`? I don't think that will cause a test failure if we first check if `this.props.field.value` is not blank?